### PR TITLE
Import console and db tasks from template + add specs

### PR DESCRIPTION
**Pull request by solnic**

### CODE REVIEW

File: lib/hanami/cli/commands/monolith/console.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      module Monolith
+        class Console < Application
+          desc "Application REPL"
+
+          def call(**_opts)
+            require "pry"
+
+            prompt = application_prompt
+
+            Pry.config.prompt = Pry::Prompt.new(
```
Comment (jodosha) [inline lib/hanami/cli/commands/monolith/console.rb:19]: @solnic why `pry` instead of IRB, which is part of Ruby distribution?

Comment (solnic) [reply lib/hanami/cli/commands/monolith/console.rb:19]: @jodosha I haven't figured out how to set its prompt 😆 I'll give this another shot

Comment (jodosha) [reply lib/hanami/cli/commands/monolith/console.rb:19]: @solnic https://docs.ruby-lang.org/en/2.2.0/IRB.html#module-IRB-label-Customizing+the+IRB+Prompt

Comment (solnic) [reply lib/hanami/cli/commands/monolith/console.rb]: @jodosha I spent almost two hours trying to figure out how to start IRB and gave up :(

File: lib/hanami/cli/commands/application.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              require "hanami/init"
+              Hanami.application
+            end
+        end
+
+        def run_command(klass, *args)
+          klass.new(
+            out: out,
+            inflector: application.inflector,
+            fs: Dry::Files
```
Comment (jodosha) [inline lib/hanami/cli/commands/application.rb:37]: @solnic Shouldn't this be `Dry::Files.new`?

**Review by jodosha — APPROVED** [review]
@solnic LGTM, but I haven't tried manually. Left one comment as clarification.

File: lib/hanami/cli/commands/application.rb (hunk 2 above)
Comment (solnic) [reply lib/hanami/cli/commands/application.rb:37]: @jodosha probably 🙂 it's not used yet (generators are disabled). I'll address this separately.

**Pull request merged**